contracts/src/v0.8/keystone/KeystoneForwarder.sol

```diff
@@ -214,14 +209,13 @@ contract KeystoneForwarder is IForwarder, ConfirmedOwner, TypeAndVersionInterfac
     return keccak256(bytes.concat(bytes20(uint160(receiver)), workflowExecutionId, reportId));
   }
 
-  // get transmitter of a given report or 0x0 if it wasn't transmitted yet
   function getTransmitter(
```

Comment:
Should this be moved to router?

Reply:
I kept it as a wrapper since it calculates the combined ID for you. Would be easier to use offchain